HackingWithSwift/week1/day_5/Writing funktions/Writing functions.playground/Contents.swift

```diff
@@ -0,0 +1,21 @@
+func printHelp () {
+    let message = """
+You need turn on geolocation in the settings
+"""
+    print(message)
+}
+printHelp()
+
+func dayForGym () {
+    print("Monday, Tuesday, Friday")
+    print("Sunday and Saturday")
+}
+dayForGym()
+
+func PaymantError () {
+    let message = """
+Try again latter
+"""
+    print(message)
+}
+PaymantError()
```

Comment:
Please remember the naming convention for Swift functions. Function names should start with a lowercase letter. I suggest renaming PaymantError to paymentError:

```
func paymentError() {
    let message = """
    Try again later
    """
    print(message)
}
paymentError()
```